connectors/google/google-gemini/src/test/java/io/camunda/connector/gemini/mapper/PromptsMapperTest.java

```diff
@@ -56,16 +56,6 @@ void mapNullEntryShouldThrowEx() throws Exception {
     assertThat(ex).hasMessage(EMPTY_PROMPT_MSG);
   }
 
-  @Test
-  void mapWithWrongValueTypeShouldThrowEx() throws Exception {
-    List<Object> input =
-        readValue("src/test/resources/prompts_with_wrong_value_type.json", List.class);
-
-    IllegalArgumentException ex =
-        assertThrows(IllegalArgumentException.class, () -> promptsMapper.map(input));
-    assertThat(ex).hasMessage(INVALID_PROMPT_MSG_FORMAT.formatted(Map.of(TEXT_KEY, List.of(1))));
-  }
-
```

Comment:
Should we keep this test case but use a different type of input data?

Reply:
Good question, I removed it as it tested something that is supported work now. There are additional tests that cover other illegal prompt input so I think its fine to just remove this.